odim-controller/helmcharts/odimra-config/templates/configmaps.yaml

```diff
@@ -75,7 +75,14 @@ data:
     		"SystemCollection": [
     			"Chassis",
     			"LogServices",
-    			"WorkloadPerformanceAdvisor"
+                        "Managers",
```

Comment:
Why skip managers under systems ?

Reply:
Chassis and managers are already handled in a separate process, this list is added for ignoring the contents to be added in DB under system